src/foam/nanos/auth/User.js

```diff
@@ -33,8 +33,9 @@ foam.CLASS({
     'foam.util.SafetyUtil',
     'static foam.mlang.MLang.EQ'
   ],
-
-  documentation: '',
+  
+  documentation: `Authenticates a user in the system. 
```

Comment:
The User model/record represents a user of the system.  Has nothing to do with authentication itself.